src/Generator.h

```diff
@@ -3634,7 +3639,13 @@ class GeneratorBase : public NamesInterface, public AbstractGenerator {
     std::unique_ptr<GeneratorParamInfo> param_info_ptr;
 
     std::string generator_registered_name, generator_stub_name;
-    Pipeline pipeline;
+    Pipeline pipeline_;
```

Comment:
Why the new underscores? I thought we typically didn't do that for member variables.

Reply:
Ah, was trying to track down usage by doing the 'rename and see what fails to compile' trick. Will revert.